packages/ddp-client/livedata_connection.js

```diff
@@ -38,7 +38,11 @@ var Connection = function (url, options) {
     reloadWithOutstanding: false,
     supportedDDPVersions: DDPCommon.SUPPORTED_DDP_VERSIONS,
     retry: true,
-    respondToPings: true
+    respondToPings: true,
+    // When updates are coming within this ms interval, batch them together.
+    bufferedWritesInterval: 5,
+    // Flush buffers immediately if writes are happening continuously for more than this many ms.
+    bufferedWritesMaxAge: 500
```

Comment:
TBH; I'm not sure about this one (it was added by @mitar). Does `Websocket.onmessage` take precedence over `setTimeout` in every browser/engine?

I was going to we-write this as a maximum buffer size, rather than a temporal boundary.